src/utilities/calculator.js

```diff
@@ -162,14 +162,14 @@ const decliningRepaymentCalculatorForThreeMonths = (
   const residual = productPrice - actualDownpayment
   const normalInstallment = residual / count
   const normalInstallmentAndInterest =
-    normalInstallment + (interest / 100) * residual
+    normalInstallment + (params.interest / 100) * residual
   const totalNormalInstallmentAndInterest = normalInstallmentAndInterest * count
 
   const totalNormalInstallmentAndInterestAndDownpayment =
     totalNormalInstallmentAndInterest + actualDownpayment
 
   const labelPrice =
-    (tax / 100) * totalNormalInstallmentAndInterestAndDownpayment +
+    (params.tax / 100) * totalNormalInstallmentAndInterestAndDownpayment +
```

Comment:
Why are we having calc changes here?

Reply:
TaoFeek saw a bug and he fixed it during a call with him

Reply:
Should I remove or just leave it